poly-synth/index.html

```diff
@@ -20,23 +20,55 @@
     <script src="src/poly-synth-demo.js"></script>
     <script src="src/poly-synth.js"></script>
     <script src="src/poly-synth-voice.js"></script>
+    <style>
+      #container {
+        display: flex;
+        flex-direction: row;
+      }
+      .controls {
+        border-style: solid;
+        margin: 5px;
+      }
+    </style>
   </head>
   <body>
+    <h1>Polyphonic Subtractive Synthesizer</h1>
     <p>
-     The key A on your computer keyboard maps onto the note C, and the letters
-     to the right of A (SDFGHJ) map onto the notes above C in a C major scale
-     (DEFGAB). The letters WETYU map onto the black keys of a piano.
+      The key A on your computer keyboard maps onto the note C, and the letters
+      to the right of A (SDFGHJ) map onto the notes above C in a C major scale
+      (DEFGAB). The letters WETYU map onto the black keys of a piano.
+    </p>
+    <p>
+      The synthesizer constructs voices according to the volume and filter ADSR
+      settings. The volume ADSR modulates the gain of the signal over time, and
+      the filter ADSR modulates the detune of the cutoff filter, meaning the
+      cutoff specifies the filter's starting point and it will detune over
+      time depending on the ADSR settings. A detune amount of 1 means the filter
+      will deviate by at most 2400 cents from the filter cutoff value. For both
+      the volume and filter ADSR,
+      <ul>
```

Comment:
I find this description is too verbose and confusing. Just having 3~4 bullet points about how to use the synth would be good enough. You don't need to explain how synthesizer works here.